sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -49,6 +49,17 @@ enum BuildState { BS_InProgress, BS_Done, BS_Failed };
 
 static constexpr char UseSpvEnv[]("SYCL_USE_KERNEL_SPV");
 
+/// This function enables ITT annotations in SPIR-V module by setting
+/// a specialization constant if INTEL_LIBITTNOTIFY64 env variable is set.
```

Comment:
You mean INTEL_ENABLE_OFFLOAD_ANNOTATIONS, right?

Reply:
That's right, I forgot to update the comment.